contrib/init/datum_gateway.service

```diff
@@ -0,0 +1,89 @@
+[Unit]
+Description=DATUM Gateway
+Documentation=https://github.com/OCEAN-xyz/datum_gateway
+
+#
+# Expected file locations when using hardening options:
+# binary: /usr/bin/datum/
+# configuration: /etc/datum/
+# logs: /var/lib/datum_gateway/
+#
+
+# https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
+Wants=network-online.target
+After=network-online.target
+# alternatively, wait until bitcoind is up
+#After=bitcoind.service
+
+[Service]
+
+ExecStart=/usr/bin/datum/datum_gateway --config=/etc/datum/datum_gateway_config.json
+
+# Make sure the config directory is readable by the service user
+PermissionsStartOnly=true
+ExecStartPre=/bin/chgrp datum /etc/datum
+
+#
+# Process Management
+#
+
+Type=simple
+NotifyAccess=all
+PIDFile=/run/datum/datum_gateway.pid
```

Comment:
Does systemd create the directory for us?

Reply:
Yes, systemd dynamically creates the directory on start and then deletes it on stop.